src/main/webapp/nickname.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+      <h1>Black Student Insight</h1></a>
+    </header>
+    <main>
+      <h2>Enter a Name to Appear on All of Your Reviews</h2>
+      <form method="POST" action="/user">
```

Comment:
Wow, awesome, so we allow users to set their nicknames with us? I thought we planned to just use the nick name from the [user object](https://cloud.google.com/appengine/docs/standard/python/refdocs/google.appengine.api.users#google.appengine.api.users.User) returned from the user api? But I am fine with the solution that allows a user to set its own nick name on the website.

Reply:
Yea I tested that and it just leaves the google address sometimes so I felt like this would be cooler haha